SUPPORT.md

```diff
@@ -4,14 +4,11 @@ Looking for help with Pony? Awesome. We are here to help. Before you open an iss
 
 ## Should I open an issue?
 
-We ask you to refrain from opening GitHub issues for non-actionable items. We use GitHub issues to track bugs and other problems that can be resolved. If you are not sure how to code a particular problem, how to decipher an error message, wondering about a benchmark, or anything of that sort, please use the [mailing list][mailing list] or [Zulip][zulip].
+We ask you to refrain from opening GitHub issues for non-actionable items. We use GitHub issues to track bugs and other problems that can be resolved. If you are not sure how to code a particular problem, how to decipher an error message, wondering about a benchmark, or anything of that sort, please use reach out via [our Zulip community][zulip].
 
 ## How to get help
 
-You have a couple of different options on how you can get help from the Pony community. If you are looking for an answer "right now," we suggest you give our Zulip community a try. If you ask a question, be sure to hang around until you get an answer. If you don't get one, or Zulip isn't your thing, we have a friendly mailing list you can try. Whatever your question is, it isn't dumb, and we won't get annoyed.
-
-* [Mailing list][mailing list].
-* [Zulip][zulip].
+If you are looking for an answer "right now," we suggest you give our [Zulip community][zulip-beginner-help] a try. Whatever your question is, it isn't dumb, and we won't get annoyed.
```

Comment:
Does this still need the "right now" qualifier, since there is no longer a distinction between live chat and mailing list?

Reply:
Hmmm...

Reply:
Yeah I think it does, looking at the larger context.